framework/platform/platform.cpp

```diff
@@ -248,7 +248,7 @@ void Platform::terminate(ExitCode code)
 	// Halt on all unsuccessful exit codes unless ForceClose is in use
 	if (code != ExitCode::Success && !using_plugin<::plugins::ForceClose>())
 	{
-#ifndef ANDROID
+#ifdef PLATFORM__WINDOWS
 		std::cout << "Press return to continue";
 		std::cin.get();
 #endif
```

Comment:
Maybe the complete if-clause, not only the body, should be PLATFORM__WINDOWS-specific?

Reply:
Done